react-components/config/webpack.config.prod.js

```diff
@@ -260,23 +261,23 @@ module.exports = {
     // Otherwise React will be compiled in the very slow development mode.
     new webpack.DefinePlugin(env.stringified),
     // Minify the code.
-    new webpack.optimize.UglifyJsPlugin({
-      compress: {
-        warnings: false,
-        // Disabled because of an issue with Uglify breaking seemingly valid code:
-        // https://github.com/facebookincubator/create-react-app/issues/2376
-        // Pending further investigation:
-        // https://github.com/mishoo/UglifyJS2/issues/2011
-        comparisons: false,
-      },
-      output: {
-        comments: false,
-        // Turned on because emoji and regex is not minified properly using default
-        // https://github.com/facebookincubator/create-react-app/issues/2488
-        ascii_only: true,
-      },
-      sourceMap: shouldUseSourceMap,
-    }),
+    //new webpack.optimize.UglifyJsPlugin({
+    //  compress: {
+    //    warnings: false,
+    //    // Disabled because of an issue with Uglify breaking seemingly valid code:
+    //    // https://github.com/facebookincubator/create-react-app/issues/2376
+    //    // Pending further investigation:
+    //    // https://github.com/mishoo/UglifyJS2/issues/2011
+    //    comparisons: false,
+    //  },
+    //  output: {
+    //    comments: false,
+    //    // Turned on because emoji and regex is not minified properly using default
+    //    // https://github.com/facebookincubator/create-react-app/issues/2488
+    //    ascii_only: true,
+    //  },
+    //  sourceMap: shouldUseSourceMap,
+    //}),
     // Note: this won't work without ExtractTextPlugin.extract(..) in `loaders`.
```

Comment:
Oops I missed this in rebase I guess.

Reply:
This would be fixed by a new `rebase`: https://github.com/cyverse-de/ui/pull/289#discussion_r184244301